docs/create_usb_boot_disks.rst

```diff
@@ -0,0 +1,214 @@
+#. The *Admin Workstation* Tails USB.
+
+.. tip:: This process will take some time, most of which will be spent
+	 waiting around.
+
+Enable Persistent Storage
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, everything you save while running Tails will be securely
+erased and discarded when you power off or reboot your system.
+
+Because we will need to keep certain settings and files saved between
+sessions, you will need to enable persistence.
+
+Creating an encrypted persistent volume will allow you to securely save
+information and settings in the free space that is left on your Tails
+drive. You will need to create a persistent storage on each Tails drive. Each
+drive's persistent storage partition needs its own unique, complex 
+passphrase that's easy to write down or remember. 
+
+For instructions on how to generate a strong passphrase, see :doc:`the 
+Passphrases page <passphrases>`.
+
+Please use the instructions on the `Tails website
+<https://tails.boum.org/doc/first_steps/persistence/index.en.html>`__
+to make the persistent volume on each Tails drive you create. When
+creating the persistence volume, you will be asked to select from a
+list of features, such as 'Personal Data'. You should enable **all** features by
+selecting each item in the list.
+
+Some other things to keep in mind:
+
+-  Right now, you need to create a persistent volume on both the *Admin
+   Workstation* Tails drive and the *Secure Viewing Station* Tails
+   drive.
+
+-  Each journalist will need their own Tails drive with their own
+   persistent volume secured with their own passphrase — but :doc:`that comes
+   later <onboarding_journalists>`.
+   
+.. note:: Tails doesn't always completely shut down and reboot
+	  properly when you click "restart", so if you notice a
+	  significant delay, you may have to manually power off and
+	  restart your computer for it to work properly.
+
+.. warning:: Make sure that you never use the *Secure Viewing Station*
+	     Tails drive on a computer connected to the Internet or a
+	     local network. This Tails drive will only be used on the
+	     air-gapped *Secure Viewing Station*.
+   
+  
+Ubuntu Introduction
+-------------------
+
+.. note:: Installing Ubuntu is simple and may even be something you are very familiar
+  with, but it is **strongly** encouraged that you read and follow this documentation
+  exactly as there are some "gotchas" that may cause your SecureDrop setup to break.
+
+The SecureDrop *Application Server* and *Monitor Server* run **Ubuntu Server
+20.04.4 LTS (Focal Fossa)**. To install Ubuntu on the servers, you must first
+download and verify the Ubuntu installation media.
+
+.. _download_ubuntu:
+
+Download the Ubuntu Installation Media
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The installation media and the files required to verify it are available on the
+`Ubuntu Releases page`_. You will need to download the following files:
+
+* `ubuntu-20.04.4-live-server-amd64.iso`_
```

Comment:
Heads up: It looks like 20.04.5 is around the corner (September 1 according to https://wiki.ubuntu.com/Releases). We'll want to point to that version consistently once it's live.

Reply:
Just a note that although it's not showing up there in the Wiki yet, Ubuntu 20.04.5 was released earlier today. The links are now active, so I have updated everything.